diesel/src/mysql/types/mod.rs

```diff
@@ -12,6 +12,12 @@ use mysql::Mysql;
 use serialize::{self, IsNull, Output, ToSql};
 use sql_types;
 
+impl sql_types::HasSqlType<sql_types::Unsigned<sql_types::Integer>> for Mysql {
```

Comment:
This needs to support more than just `Integer` doesn't it?

Reply:
Yep, is it just `SmallInt`, `Integer`, and `BigInt`?

Reply:
This impl can probably just be generic over everything.

Reply:
Unsigned strings let's go ! :trollface:

Reply:
There wouldn't be a corresponding `AsExpression` impl, so... :P 

There's nothing to prevent you from putting `Unsigned<Text>` in your table declaration already